CONTRIBUTING.md

```diff
@@ -259,12 +259,16 @@ Once you are satisfied with the title, description and file changes included, su
 If this is your first contribution to the DSC Resource Kit, you may be asked to sign a [Contribution Licensing Agreement](https://cla.microsoft.com/) (CLA) before your changes can be reviewed:
 ![GitHubCLARequired.png](Images/GitHubCLARequired.png)
 
-Once you sign the CLA, the Microsoft CLA bot will automatically update your PR:
+Once you sign the CLA, the Microsoft CLA bot will automatically update the comment in your PR:
 ![GitHubCLASigned.png](Images/GitHubCLASigned.png)
 
+The CLA status check should also pass in your PR.
+![GitHubCLASigned.png](Images/GitHubCLAStatusCheck.png)
```

Comment:
This pic doesn't look like it's rendering. Not sure if it's because the pic is new or because the name in the square brackets needs to be changed too?

Reply:
I fixed the alt text - but it rendered okay for me in both Safari and Chrome before that. Can you screen shot how it looks like for you? Maybe it's the picture that is wrong.